### ISSUE TITLE
Could not install PHP when installing Nginx

### ISSUE THREAD

**Issue by sargunv** [liaralabs/swizzin#49]
```
Installing nginx
E: The repository 'https://downloads.plex.tv/repo/deb ./public Release' does not have a Release file.
E: The repository 'http://download.opensuse.org/repositories/home:/emby/xUbuntu_  Release' does not have a Release file.
ERROR: APT-GET could not install a required package: php7.0-fpm. That's probably not good...
ERROR: APT-GET could not install a required package: php7.0-cli. That's probably not good...
ERROR: APT-GET could not install a required package: php7.0-dev. That's probably not good...
ERROR: APT-GET could not install a required package: php7.0-xml. That's probably not good...
ERROR: APT-GET could not install a required package: php7.0-curl. That's probably not good...
ERROR: APT-GET could not install a required package: php7.0-xmlrpc. That's probably not good...
ERROR: APT-GET could not install a required package: php7.0-json. That's probably not good...
ERROR: APT-GET could not install a required package: php7.0-mcrypt. That's probably not good...
ERROR: APT-GET could not install a required package: php7.0-mbstring. That's probably not good...
ERROR: APT-GET could not install a required package: php7.0-opcache. That's probably not good...
sed: can't read /etc/php/7.0/fpm/php.ini: No such file or directory
```

This is on Ubuntu Server 17.10.

**Comment by liaralabs** [comment]
It looks like 17.10 is dropping php7.0 in favour of 7.1.

I'm currently hammering out some details of the fix and should have something pushed by today.

**Comment by liaralabs** [comment]
I have pushed a fix to the develop branch. Since this is a fairly comprehensive commit, I need to test all conceivable scenarios to ensure its behaving as it should.

I ran out of time at this particular moment, but will have time soon to get this pushed to master

**Comment by LaMpiR** [comment]
Have the same problem. Thank you.

**Comment by liaralabs** [comment]
Fix pushed to master in 4667b40b43d204a150afadc645bd2dfc1231b46d
